spec/fluent.ebnf

```diff
@@ -43,15 +43,15 @@ message              ::= identifier (value attribute* | attribute+)
 
 value                ::= _? '=' __? pattern
 pattern              ::= (text | placeable)+
-/* text can only include newlines if they're followed by an indent */
+
```

Comment:
meh, cannot comment on the line below, but ... no ` and `

Reply:
Do you mean no `and` in `\ and { must be escaped`? `\` and `{` correspond to `#x5c#x7b` which this rule says must be escaped.

Reply:
oh goodness, yeah, that comment is really hard to read. Sorry. `\` just slipped my mind here.